### PR TITLE
Nerfs blobbernaut creation, tweaks blob reagents

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
wip: Blobbernaut creation costs 30 points, and does much more damage to the factory.
rscdel: Blob factories regenerate at half normal rate.
tweak: Blob reagents tweaked;
rscadd: Ripping Tendrils does slightly more brute damage, but less stamina damage.
rscdel: Lexorin Jelly does less brute damage.
rscadd: Energized Fibers does slightly more burn damage.
rscadd: Sporing Pods does slightly more toxin damage.
rscadd: Replicating Foam will try to replicate when hit more often.
rscadd: Hallucinogenic Nectar does slightly more toxin damage and causes hallucinations for longer.
rscadd: Cryogenic Liquid does more burn damage.
experiment: Synchronous Mesh does slightly less damage with one blob but massive damage with more than one nearby blob.
rscdel: Dark Matter does less brute damage.
rscdel: Sorium does slightly less brute damage.
rscdel: Pressurized Slime has a lower chance to emit water when killed and when attacking targets.
/:cl:

The blobbernaut nerfs are all to creation:
Creating a blobbernaut costs 50% more resources.
The factory used takes 33% more damage, 80% of its maximum health.
Factories regenerate at half the rate they did before, meaning there's a longer delay between them being able to be used for blobbernauts.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
YOU ASKED FOR NERFS, AND HERE WE ARE.

**Comment by PKPenguin321** [comment]
i still think we should just roll back blob to before blobbernauts were sentient, when blobs could self pop (maybe on this one), and collective hivemind was a thing instead of weird scaling

**Comment by Zilenan91** [comment]
@ChangelingRain Pls nerf slime blobbernauts. They spew water on every hit, and if you walk they're faster than you, and they can drag corpses and hit them to make water everywhere. They're too powerful and kill entire stations.

**Comment by ChangelingRain** [comment]
I did, read the chem changes.

**Comment by Zilenan91** [comment]
Oh wow I thought that only applied to blob tiles not blobbernauts. So it used to be 90% and now it's a 50% chance to emit water? I still think that's too good tbh, it gives blob a stun and that's still really bad. You should just trash their pissing powers completely and make them do more damage.

**Comment by ChangelingRain** [comment]
Blobbernauts have a 17% chance to make water when hitting, while the blob itself has a 25% chance.
The damage reaction is blob only, but the chem reactions aren't.

**Comment by Zilenan91** [comment]
Oh ok that's fine then.

**Comment by neersighted** [comment]
```
Would YOU like to play as Leporzine Slime Blobnaut?
[YES] [NO] [OH MY GOD POPUPS]
```

**Comment by ChangelingRain** [comment]
tgui popups when

**Comment by neersighted** [comment]
Soon (tm).

File: code/game/gamemodes/blob/powers.dm [hunk 1]
```diff
@@ -76,20 +76,20 @@
...
-	set name = "Create Blobbernaut (20)"
+	set name = "Create Blobbernaut (30)"
 	set desc = "Create a powerful blobbernaut which is mildly smart and will attack enemies."
 	var/turf/T = get_turf(src)
 	var/obj/effect/blob/factory/B = locate(/obj/effect/blob/factory) in T
 	if(!B)
 		src << "<span class='warning'>You must be on a factory blob!</span>"
 		return
-	if(B.health < B.maxhealth*0.6) //if it's at less than 60% of its health, you can't blobbernaut it
+	if(B.health < B.maxhealth*0.8) //if it's at less than 60% of its health, you can't blobbernaut it
```
Comment (duncathan) [inline code/game/gamemodes/blob/powers.dm]: Update comment

**Comment by tkdrg** [comment]
sure whatever

File: code/game/gamemodes/blob/powers.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/blob/powers.dm]: I'LL FIX THIS LATER(:tm:)

**Pull request merged**